pkg/executor/executortype/poolmgr/funchandlers.go

```diff
@@ -37,7 +37,7 @@ func getIstioServiceLabels(fnName string) map[string]string {
 	}
 }
 
-func (gpm *GenericPoolManager) FunctionEventHandlers(kubernetesClient *kubernetes.Clientset, fissionfnNamespace string, istioEnabled bool) k8sCache.ResourceEventHandlerFuncs {
+func FunctionEventHandlers(logger *zap.Logger, kubernetesClient *kubernetes.Clientset, fissionfnNamespace string, istioEnabled bool) k8sCache.ResourceEventHandlerFuncs {
```

Comment:
Since we are trying to improve the code, I believe it would be nice to have comments for exported functions.